src/transforms/branch.ts

```diff
@@ -50,14 +56,38 @@ export default async (github: GitHubAPI, webhookPayload: WebhookPayloadCreate) =
 
 	const lastCommit = await getLastCommit(github, webhookPayload, issueKeys);
 
+	console.log("SHOW ME THE RETURN");
```

Comment:
Is this from another change? Doesn't seem to have to do with the config as code stuff.